build.xml

```diff
@@ -662,32 +662,6 @@
         </if>
     </target>
 
-    <target name="timezones">
```

Comment:
I think we will need to discuss viable dev solutions for the problem you describe.